### DISCUSSION TITLE
身披天赋铠甲的二舅 侥幸过着普通人的一生

### DISCUSSION THREAD

**Discussion by frshgrass** [WeNeedHome/SummaryOfLoanSuspension#1034] [General]
二舅能这样做是因为：
       1，超高智商，学东西快，有很多技能借以糊口和才受村民尊重（是因为你有用才能这样，在农村生活都知道，农村很丛林法则的，你没本事，家族大人多会欺负你到死）。这种智商天赋我说他千中无一不过分吧。

 2，二舅面对苦难，能感受到他的坚强意志和乐观。但是乐观努力意志坚强这类情商也是与生俱来的天赋（其实努力和意志算不算天赋，之前知乎讨论过，很多人现身说法就是，部分人觉得坚持和努力并不辛苦，他们天生就这样，也喜欢这样的状态）。那这种情商我说他万中无一不过分吧

所以呢，二舅在情商和智商的双重天赋下，在面对苦难的漫天炮火，天赋自带的高情商和智商犹如给二舅穿上了厚厚的防弹盔甲，即使这样也是过着普通人的生活。

那我们呢 智商和情商天赋一般的人  明知道有坏人在制造苦难，你觉得你的情商和智商比得过二舅吗 身披草甲平凡的我们面对炮火直接几下就无了。（我想过我要是遇到二舅这样的苦难，我的智商和心性 要不抑郁自杀 要不变成村口树先生 疯掉）

二舅的智商和情商天赋所在万中无一，也选择不娶妻孑然一身，才可以在厚厚的铠甲下迎面炮火。

但是世界上还有很多生下来就像我们这样读书永远是中流或者拼熬夜拼辛苦，然后上有老下有小各方面压力也大，心思沉重敏感的平凡人 。我们如果独自面对炮火直接无了。

 我们的孩子大部分也是情商和智商的普通人，我们还自己把最好的一面展现，然后教育要善良让孩子单纯得像天使一样。你敢让我们的孩子面对这样的炮火吗？

应该团结起来 砸烂苦难的制造者  我们要接受自己只是普通人  普通人要在没有恶人和炮火世界开心得生活。

万中无一的，集天赋决定的情商和智商的，幸存者偏差。我们只是天赋一般的普通人，如果信了这套话术包装的鸡汤，赤身裸体的接受苦难的炮火，你将慢慢的血肉横飞无掉了。

无非是改头换面的于丹讲论语，话术更自然而已（呵呵 也不知道 火了之后 二舅的残疾证还能不能办下来）

苦难要咬紧牙关 但从来不值得歌颂（失职的地方政府 我也不知道 他哪来的脸宣传这些）

**Comment by toxin08x** [comment]
https://www.youtube.com/watch?v=IZ2WgERJ_IY
他讲的二舅解读就挺有意思的

**Comment by TALK1949** [comment]
二舅什么梗？

**Comment by Skyjc2020** [comment]
一只无形的大手一直都在，怎么挣扎都无法逃脱，最后～

**Comment by JavaCoder-Lambert** [comment]
然后呢？周劼打破幻想

**Comment by shawn-yee** [comment]
普通人走二舅那条路早自杀了，地狱模式，玩个毛。
但如果在美国，二舅早就被赔偿成富翁了。

**Reply by toxin08x** [reply]
如果在美国，二舅会活的非常非常好... 美国对残疾人的照顾比中国要健全得多。并且二舅蓝领的这些天赋在美国非常非常赚钱... 我家一年多前低温爆了3处水管找了个老中师傅超级友情价修还花了800刀。正常市场价基本上得1500+

**Comment by fireflyhoo** [comment]
都是韭菜, 这个韭菜比较坚强而已, 所以赞美坚强的韭菜咯?
